Scripts/update-translations.rb

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env ruby
-# encoding: utf-8
+# frozen_string_literal: true
```

Comment:
🤔 I wonder what will happen without this encoding annotation. Probably nothing. And anyways, this approach to localizations is deprecated. If it broke, it would be good excuse to upgrade the setup.

Reply:
I believe `utf-8` is the default, so it is redundant to declare it.